newm-server/src/main/kotlin/io/projectnewm/server/di/DependencyInjectionInstall.kt

```diff
@@ -24,7 +25,8 @@ fun Application.installDependencyInjection() {
             userKoinModule,
             authKoinModule,
             songKoinModule,
-            playlistKoinModule
+            playlistKoinModule,
+            cloudinaryKoinModule
```

Comment:
I've been in the habit of adding a trailing comma to any param lists in Kotlin. That way, when you add something additional to the list, the diff only shows the single line you added.